### PR TITLE
Add tests for timezone

### PR DESCRIPTION
**Pull request by perlpunk**
After #163, this adds some test data to check if the datetime objects
return the correct timezone

### CODE REVIEW

File: tests/lib3/test_constructor.py [hunk 1]
```diff
@@ -38,7 +38,19 @@ def construct1(constructor, node):
...
+    def my_time_constructor(constructor, node):
+        seq = constructor.construct_sequence(node)
+        dt = seq[0]
+        tz = None
+        try:
+            tz = dt.tzinfo.tzname(dt)
+        except:
+            pass
+        mytime = [dt, tz]
+        return mytime
```
Comment (akaIDIOT) [inline tests/lib3/test_constructor.py]: Anything against `return [dt, tz]`?

File: tests/lib/test_constructor.py [hunk 2]
```diff
@@ -41,7 +41,20 @@ def construct1(constructor, node):
     def represent1(representer, native):
         return representer.represent_mapping("!tag1", native.__dict__)
 
+    def my_time_constructor(constructor, node):
+        seq = constructor.construct_sequence(node)
+        dt = seq[0]
+        tz = None
+        try:
+            tz = dt.tzinfo.tzname(dt)
+            tz = 'UTC' + tz
```
Comment (akaIDIOT) [inline tests/lib/test_constructor.py]: Hmm, does the default `datetime.timezone` implementation include the `UTC` prefix by default for python 3? Changing the `__str__` for the `timezone` helper for python 2 to include `UTC` here might make more sense to keep those aligned.

**Comment by akaIDIOT** [comment]
Thanks for taking this effort :D Don't see any issues without, commented on some minor niggles, but those might be perfectly in line with pyyaml's code style, I'll leave that up to you :)

File: tests/lib/test_constructor.py (hunk 2 above)
Comment (perlpunk) [reply tests/lib/test_constructor.py]: I was wondering about that, too.
Since we expect it to be UTC (plus offset) always I guess I can add that.

**Comment by perlpunk** [comment]
thanks @akaIDIOT :)

**Comment by perlpunk** [comment]
released https://pypi.org/project/PyYAML/5.3/

**Pull request merged**